specification/under_development/output_schemas/Labeled_And_Async.md

```diff
@@ -1,2 +1,238 @@
 # Labeled Format for Asynchronous Output Generation
 
+The labeled format for asynchronous output generation is the same as the [ordered format][] with the following changes:
```

Comment:
An additional paragraph describing the role of the labels for asynchronous output generation would make it more clear to the reader why labels are needed.

We could also include a recommendation that the consumer of the output format should be the one setting the labels and the reasoning behind it would be a nice addition.